modules/cockroachdb/cockroachdb.go

```diff
@@ -233,6 +239,46 @@ func addTLS(ctx context.Context, container testcontainers.Container, opts option
 	return nil
 }
 
+// setRecommendedSettings applies the cluster settings recommended by cockroachlabs for testing clusters.
```

Comment:
question: would it make sense to make this a function option e.g. `WithClusterSettings(settings ...string)` where settings defaults to the below if nil?